GeneratorInterface/GenFilters/src/PythiaMomDauFilter.cc

```diff
@@ -0,0 +1,161 @@
+
```

Comment:
@vmariani  - why this is not extending an existing filter with added functionality?